packages/module/src/components/contextmenu/ContextMenu.tsx

```diff
@@ -31,30 +28,11 @@ const ContextMenu: React.FunctionComponent<ContextMenuProps> = ({
 
   return (
     <Popper {...other} closeOnEsc closeOnOutsideClick open={isOpen} onRequestClose={handleOnRequestClose}>
-      <DropdownContextDeprecated.Provider
-        value={{
-          onSelect: handleOnRequestClose,
-          toggleTextClass: styles.dropdownToggleText,
-          toggleIconClass: styles.dropdownToggleIcon,
-          menuClass: styles.dropdownMenu,
-          itemClass: styles.dropdownMenuItem,
-          toggleClass: styles.dropdownToggle,
-          baseClass: styles.dropdown,
-          baseComponent: 'div',
-          sectionClass: styles.dropdownGroup,
-          sectionTitleClass: styles.dropdownGroupTitle,
-          sectionComponent: 'section',
-          disabledClass: styles.modifiers.disabled
-          // hoverClass: styles.modifiers.hover,
-          // separatorClass: styles.dropdownSeparator,
-        }}
-      >
-        <div className={css(styles.dropdown, styles.modifiers.expanded)}>
-          <DropdownMenuDeprecated className={css(topologyStyles.topologyContextMenuCDropdownMenu)} autoFocus>
-            {children}
-          </DropdownMenuDeprecated>
-        </div>
-      </DropdownContextDeprecated.Provider>
+      <div className={css(styles.dropdown, styles.modifiers.expanded)}>
+        <Dropdown toggle={() => <></>} className={css(topologyStyles.topologyContextMenuCDropdownMenu)}>
```

Comment:
I'm noticing that the old dropdowns in the context menu demo would close after you select an item. IDK if that's a built in behavior or a quirk of the demo. Just checking it was an intended change?

Reply:
IMO, they shouldn't close when checking a checkbox so as to allow the user to select multiple w/o reopening. Not sure why the v4 version closed on each selection but I like it better this way.

Reply:
Thank you @nicolethoen , good find. Updated